collect_app/src/main/res/layout/quit_form_dialog_layout.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/margin_large">
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/save_explanation"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/save_explanation"
+        android:textAppearance="?textAppearanceBodyMedium"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <com.google.android.material.button.MaterialButton
```

Comment:
In the main menu, the gray buttons have a gray ripple effect. SHouldn't it be the same here?

Reply:
We're just using a standard Material 3 style here so they should be using whatever the default is (within our theme colors). I'll double-check if the primary ripple we're seeing is incorrect. It could be that we're missing a Material 3 color attribute.

Reply:
The Main Menu buttons are "Elevated" rather than "Outlined" so it actually makes sense that they might have different ripple colors. However, the docs for Material Components [suggest the ripple color should be `colorOnSurface`](https://github.com/material-components/material-components-android/blob/master/docs/components/Button.md#outlined-button).

Digging into this, it seems like the color being used ends up being primary. I tried switching our theme to `Theme.Material3.DayNight.NoActionBar` and saw the same thing, and looking into the implementation it semed to be that the [ripple was defined by `colorPrimary`](https://github.com/material-components/material-components-android/blob/76711889df757f2ef7e5b7d8f906390828495d16/lib/java/com/google/android/material/button/res/values/tokens.xml#L26). This could be a bug in Material Components, or it could be that the docs are out of date. Either way, I don't think we should make any changes here as the spec itself doesn't mention ripple colors, so I think we can regard whatever Material Components does as gospel.